src/components/Single-review.jsx

```diff
@@ -1,25 +1,33 @@
 import { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
-import { getSingleReview } from './api';
+import { getSingleReview, getCommentById } from './api';
 
 export const SingleReview = () => {
 	const [review, setReview] = useState([]);
+	const [comments, setComments] = useState([]);
 	let { review_id } = useParams();
 	const [isLoading, setIsLoading] = useState(true);
 	const [error, setError] = useState(null);
+	const [isComments, setIsComments] = useState(false);
```

Comment:
These could be named slightly more semantically to show that it's tied to the visibility of the comments